mk/main.mk

```diff
@@ -476,9 +487,9 @@ endif
 endif
 
 LD_LIBRARY_PATH_ENV_HOSTDIR$(1)_T_$(2)_H_$(3) := \
-    $$(CURDIR)/$$(HLIB$(1)_H_$(3))
+    $$(CURDIR)/$$(HLIB$(1)_H_$(3)):$$(CFG_LLVM_INST_DIR_$(3))/lib
 LD_LIBRARY_PATH_ENV_TARGETDIR$(1)_T_$(2)_H_$(3) := \
-    $$(CURDIR)/$$(TLIB1_T_$(2)_H_$(CFG_BUILD))
+    $$(CURDIR)/$$(TLIB1_T_$(2)_H_$(CFG_BUILD)):$$(CFG_LLVM_INST_DIR_$(CFG_BUILD))/lib
```

Comment:
I don't think that this belongs in the targetdir var because this is only needed for the compiler itself to run